user/config_openssl.go

```diff
@@ -48,7 +48,12 @@ func (this *OpensslConfig) Check() error {
 	soPath, e := getDynPathByElf(this.Curlpath, "libssl.so")
 	if e != nil {
 		//this.logger.Printf("get bash:%s dynamic library error:%v.\n", bash, e)
-		this.Openssl = "/lib/x86_64-linux-gnu/libssl.so.1.1"
+		_, e = os.Stat(X86_BINARY_PREFIX)
+		prefix := X86_BINARY_PREFIX
+		if e != nil {
+			prefix = OTHERS_BINARY_PREFIX
+		}
+		this.Openssl = prefix + "/libssl.so.1.1"
```

Comment:
please use `filepath.Join()` to concatenate strings.

Reply:
of course, I prefer to use `git rebase -i HEAD~3` could I do it that way ?

Reply:
yes, try it.

Reply:
Done